src/frontend/components/SchedulingFields/index.tsx

```diff
@@ -0,0 +1,231 @@
+import { Box, DateInput, FormField, TextInput } from 'grommet';
+import { MarginType } from 'grommet/utils';
+import { DateTime, Duration, Settings } from 'luxon';
+import React from 'react';
+import { defineMessages, useIntl } from 'react-intl';
+
+import { Nullable } from 'utils/types';
+
+const messages = defineMessages({
+  startingAtDateTextLabel: {
+    defaultMessage: 'Starting date',
+    description:
+      'Label of the input field to add/update the start date of the scheduled event',
+    id: 'component.SchedulingFields.startingAtDateTextLabel',
+  },
+  startingAtTimeTextLabel: {
+    defaultMessage: 'Starting time',
+    description:
+      'Label of the input field to add/update the start time of the scheduled event',
+    id: 'component.SchedulingFields.startingAtTextLabel',
+  },
+  estimatedDurationTextLabel: {
+    defaultMessage: 'Estimated duration',
+    description:
+      'Label of the input field to add/update the estimated duration of the scheduled event',
+    id: 'component.SchedulingFields.estimatedDurationTextLabel',
+  },
+});
+
+// build a list of times from 09h00 to 19h30
+const currentTimeForSuggestions = Duration.fromObject({ hours: 9 });
```

Comment:
We should not have restriction on this field. The meeting can start from 00h00 to 23h30

Reply:
This list is only for suggestions, we can type any time in the field.
IMHO starting at 00h00 makes the list less practical because we would need to scroll to find mainly used times.
On the other hand, as the list is filtered as we type, maybe it can be ok…
I'll push the modification you asked for, we'll see what we decide.

Reply:
It's pushed.

Reply:
An idea could be to select by default at 9am for example to don't have to scroll all the list.

Reply:
Selecting by default is maybe not the good word. I mean when I click for the first time in the input and the list appears, 9am should be the first item and the user can scroll before and after.

Reply:
I selected 11h00, as the selected item only scrolls the list to his bottom :
![image](https://user-images.githubusercontent.com/720491/162199749-2d805584-55f9-4a59-b01c-e451f29f8608.png)
wdyt ?

Reply:
It looks like this on the iframe :
![image](https://user-images.githubusercontent.com/720491/162199942-7b5cd202-6b2d-411a-bdb4-4aaedfb9034d.png)